kuksa_databroker/createbom/bomutil/maplicensefile.py

```diff
@@ -21,6 +21,7 @@
 
 MAP = {
     "Apache-2.0": "Apache-2.0.txt.gz",
+    "BlueOak-1.0.0": "BlueOak-1.0.0.md.gz",
```

Comment:
Where is this license coming from? I see the cucumber create added below, but that is just APACHE2 or MIT according to crates.io . Is this from some transitive dependency? 

In case you know, is this a dependency that is used only during build time or is it a runtime dependency? Or only at runtime for tests?

It "looks" harmless (https://spdx.org/licenses/BlueOak-1.0.0.html) but in any case, we need to clear it with Eclipse. But first we need to know where it comes from. If you can tell us @sophokles73 , maybe @bs-jokri  can try pushing it through dash, to see whether it is maybe already cleared

Reply:
Yes, it is introduced by a transitive dependency of cucumber. As such, it is only required for building/testing but not during runtime.